termloop.go

```diff
@@ -1,8 +1,9 @@
 package termloop
 
 import (
-	"github.com/nsf/termbox-go"
 	"strings"
+
```

Comment:
Sure. Derp. They are included automatically by goimports. I always forget about it in other people's repos. Maybe I cane get vim to detect gofmt/goimports based on the path.